explainer.md

```diff
@@ -269,68 +185,94 @@ In this example, we want to add a lot of items to a list in a non-janky way.
 </ul>
 
 <script>
-// Remaining items in the list are in |remainingItems|.
-// We want to add the remaining list items asynchronously.
+// Add remaining items as locked.
 remainingItems.forEach(item => {
-  requestIdleCallback((deadline) => {
-    let itemEl = createElementForItem(item);
-    itemEl.displayLock.acquire({ timeout: Infinity, activatable: true });
-    itemsList.appendChild(itemEl);
-    // We can do expensive operations to the items without worrying about the rendering costs.
-    // After we finished all the operations we can trigger a co-operative update & commit.
-    // We might do this in a fancier way by not actually committing the element,
-    // leaving it locked and detecting when to commit by using IntersectionObservers etc,
-    // but that's out of the scope of this example :)
-    doExpensiveOperationsToEl(itemEl).then(element.updateAndCommit);
-  });
+  let itemEl = document.createElement("div");
+  itemEl.style = "contain: style layout;";
+  itemEl.displayLock.acquire({ timeout: Infinity, activatable: true, size: [100, 100] });
+  itemsList.appendChild(itemEl);
+ 
+  // We can do updates to the item's subtree without worrying about the rendering costs.
+  // After we finished all the operations we can trigger a co-operative update & commit.
+  // We might do this in a fancier way by not actually committing the element,
+  // leaving it locked and detecting when to commit by using IntersectionObservers etc,
+  // but that's out of the scope of this example :)
+  itemEl.innerText = item;
+  itemEl.appendChild(...);
+  itemEl.firstChild.style = "...";
+  doExpensiveOperationsToChildren(itemEl).then(element.updateAndCommit);
 });
 
 // Getting style/layout values within a locked subtree :
-// - If we access child.offsetTop directly, it will cause a forced synchronous update
+// - If we call getComputedStyle, etc. directly, it will cause a forced synchronous style update
 //   (see [what-forces-layout](https://gist.github.com/paulirish/5d52fb081b3570c81e3a)).
 // - We can call update() on the locked element to trigger a co-operative update and get
 //   the value after the calculations finished instead.
-function getOffsetTopForLockedElementChild(child, lockedElement) {
-  return new Promise((resolve) => {
-    lockedElement.displayLock.update().then(() => { resolve(child.offsetTop) };
-  });
-}
+lockedElement.displayLock.update().then(() => {
+  // They're free!
+  getComputedStyle(child);
+  child.offsetTop;
+};
 </script>
 ```
+### The locked state
+
+#### The locked element
+
+When its lock is acquired, the locked element itself will still be rendered and laid out normally.
+Changes to the DOM, style, layout, etc of the *locked element itself* will be applied as it would on any other normal element.
+
+##### Sizing
+
+When locked, the locked element is treated similar to as if it has [size containment](https://www.w3.org/TR/css-contain-1/#containment-size) with some differences.
```

Comment:
s/with some differences/. Specifically, the following demonstrates how the element is laid out:/

Reply:
Yeah that's a fair point, keeping it as is is fine too maybe then "with some differences, outlined below" or something like that so the reader doesn't have to wonder what differences we're talking about here